README.md

```diff
@@ -4,37 +4,29 @@ Native desktop application for [Mattermost](http://www.mattermost.org/) running
 
 Originally created as "electron-mattermost" by Yuya Ochiai. Developed using [Electron](http://electron.atom.io/).
 
-![screenshot_20](https://user-images.githubusercontent.com/29708087/35169141-65d7652a-fd29-11e7-901d-735881fb7b9d.png)
+![mm-desktop-screenshot](https://user-images.githubusercontent.com/52460000/146078917-e1ba8c1f-24e5-4613-8b4b-f3507422f4f2.png)
 
 [![Circle CI](https://circleci.com/gh/mattermost/desktop.svg?style=shield)](https://circleci.com/gh/mattermost/desktop)
 
 ## Features
 
 ### Desktop integration
-* Tabs for multiple teams across multiple servers
+* Server dropdown for access to multiple teams
+* Dedicated tabs for Channels, Boards and Playbooks
 * Desktop Notifications
-  * Windows 10: Toast
-  * Windows 7-8.1: Popup like Toast
-  * OS X: Notification Center
-  * Linux: [libnotify](http://electron.atom.io/docs/v0.36.0/tutorial/desktop-environment-integration/#linux)
 * Badges for unread channels and mentions
-* Installs as a native application
-
-### Pre-packaged
-You don't have to install any other software.
-Packages are available on the [releases page](http://github.com/mattermost/desktop/releases).
+* Deep Linking to open Mattermost links directly in the app
+* Runs in background to reduce number of open windows
 
 ## Usage
 
 ### Installation
-Detailed guides are available at [docs.mattermost.com](https://about.mattermost.com/default-desktop-app-documentation/).
-
-1. Download a file from the [downloads page](https://about.mattermost.com/downloads).
-2. Launch `Mattermost` in the unarchived folder.
-3. On the first launch, please input name and URL for your Mattermost server. For example, `myserver : https://mattermost.example.com`.
+Detailed guides are available at [docs.mattermost.com](https://docs.mattermost.com/install/desktop-app-install.html).
 
-### Quit
-Ctrl or Command + Q to quit.
+1. Download a file from the [downloads page](https://mattermost.com/download/#mattermostApps) or from the [releases page](https://github.com/mattermost/desktop/releases).
+2. Run the installer or unzip the archive.
+3. Launch Mattermost from your Applications folder, menu, or the unarchived folder.
+3. On the first launch, please enter a name and URL for your Mattermost server. For example, `myserver : https://mattermost.example.com`.
```

Comment:
We should remove "myserver :" from this. See [here](https://github.com/mattermost/desktop/pull/1802#discussion_r727789116)